src/vaadin-rich-text-editor.html

```diff
@@ -1002,6 +1042,66 @@
           this.__oldDisabled = disabled;
         }
 
+        /** @private */
+        _convertLists(content) {
+          const textNode = (str) => document.createTextNode(str);
+          const tempEl = document.createElement('div');
+          tempEl.setAttribute('style', 'display: none;');
+          tempEl.innerHTML = content;
+
+          ['ul', 'ol'].forEach((type) => {
+            const startTag = `-start-${type}-`;
+            const endTag = `-end-${type}-`;
```

Comment:
This only allows nesting lists of the same, i.e. not possible to nest `<ul>` lists inside `<ol>` and vice versa.

The problem is how we detect which type of list does the `<li>` element belong to. 
Quill master branch uses `li[data-list=ordered]` and `li[data-list=bullet]`, maybe we could do the same.

Then we would need to take that into account when converting lists.

Reply:
Another thing that I noticed at Quill master branch is that they [always use `<ol>`](https://github.com/quilljs/quill/blob/e9f628b5562e78126291bb25f63d0dfc3f6c6f86/formats/list.js#L7) tag.

Then the markup for the bullet lists looks like this:

```html
<ol>
  <li data-list="ordered">0123</li>
  <li data-list="ordered">5678</li>
</ol>
```

I'm not sure if upgrading Quill to 2.0 would make this feature easier to implement or not.